content/_data/changelogs/weekly.yml

```diff
@@ -915,6 +915,47 @@
     # pull: 2960 not notable enough
     # pull: 2963 not notable enough
     # pull: 2964 not notable enough
+- version: "2.74"
+  date: 2017-08-13
+  changes:
+    - type: major rfe
+      message: >
+        Upgrade Stapler library from 1.250 to 1.252
+      references:
+        - pull: 2956
+        - url: https://github.com/stapler/stapler/blob/master/CHANGELOG.md
+          title: Stapler changelog
+    - type: rfe
+      message: >
+        Improve Polish localization.
```

Comment:
Maybe just aggregated changelog: Update Polish, Italian and Brazil localizations